.devcontainer/Dockerfile

```diff
@@ -17,6 +17,12 @@ RUN apt-get install -y python3-pip gpsd gpsd-clients python3-gps
 # Install pigpio Python library
 RUN pip3 install pigpio
 RUN pip3 install gps
+# For Camera Package
+RUN sudo apt install libopencv-dev -y
+RUN sudo apt install ros-humble-cv-bridge -y
+RUN pip install imageio-ffmpeg
+RUN pip install imageio
+

```

Comment:
does it not run once after you rebuild @MaxwellJung? If you have it as a script is it permanet?

Reply:
@dhruvna suggested adding these commands into the `dockerfile` while we were working together yesterday.

Reply:
@dhruvna made a `build-video.sh` script in `scripts/` that does the same thing. You _can_ have the commands run when building container but libopencv-dev takes forever to download so it would be pita everytime we rebuild container for whatever reason. It's better to have it be user invoked by a seperate shell script like dhruv did.